torch/_dynamo/variables/dicts.py

```diff
@@ -335,6 +335,19 @@ def call_method(
             return self.getitem_const(tx, args[0])
         elif name == "__contains__" and len(args) == 1:
             return ConstantVariable.create(args[0] in self)
+        elif name == "setdefault" and arg_hashable and self.mutable_local:
+            assert not kwargs and len(args) <= 2
```

Comment:
split into two asserts so it's easier to know which condition failed:
```suggestion
            assert not kwargs
            assert len(args) <= 2
```

Reply:
May be better to use `unimplemented`.

Reply:
It's illegal argument, so it should be assertion.

Reply:
chatted with @williamwen42 offline, unimplemented seems a more correct way, though we use assert in many other places.